pubspec.yaml

```
@@ -4,7 +4,7 @@ homepage: https://github.com/VeryGoodOpenSource/very_good_test_runner
 version: 0.1.2
 
 environment:
-  sdk: ">=2.18.0 <3.0.0"
+  sdk: ">=3.2.0 <4.0.0"
```

```suggestion
  sdk: ">=3.0.0 <4.0.0"
```

I think this has to be adjusted to `3.0.0`. The required Dart SDK version to test this package is 3.0.0 because Very Good Analysis requires it (refer to our package update policy).